extensions/interactions/base-validator.spec.ts

```diff
@@ -70,7 +70,7 @@ describe('Interaction validator', function() {
       'SubtitledHtmlObjectFactory', new SubtitledHtmlObjectFactory());
   }));
 
-  beforeEach(angular.mock.inject(function($injector, $rootScope) {
+  beforeEach(angular.mock.inject(function($injector, unused$rootScope) {
```

Comment:
@DubeySandeep what should I do with this? `$rootScope` is unused but the frontend tests are complaining about this.

Reply:
Here's the frontend test failure: https://github.com/oppia/oppia/pull/11390/checks?check_run_id=1802459472#step:7:23868